### ISSUE TITLE
Create strm for Emby as GoogleDrive-VideoStream does

### ISSUE THREAD

**Issue by chivigoku** [ddurdle/GDrive-for-KODI#188]
Morning @ddurdle 

First of all, thanks for your great Plugin.

I´ve seen there is a GoogleDrive-VideoStream which allows creations of strm with "http" instead of "plugin/".

However for those users, like me, that got an Nvidia Shield TV and want to deploy Emby-Server on it, is possible to set any config in the Kodi addon in order to create Strm´s with "http"? Then we will be able to import them to Semby-Server and reproduce movies from any Emby client.

If not, any idea to implement GoogleDrive-VideoStream on Android TV´s?

Thanks a lot in advance.
ChIvI

**Comment by ddurdle** [comment]
Yes, the plugin you referenced is a "web" version of this plugin and for precisely the purpose you mentioned.  You can even import your KODI configuration settings file to get started.

**Comment by chivigoku** [comment]
Hi @ddurdle,
 I don’t know if it’s a bit late for thinking but I didn’t get you hahaha. 

So, can I import Strm’s with describtion as “http” instead of “/plugin”? And then use it for Emby-Server

If yes, what do I need to set up in the config?

Thanks and sorry for my understanding

**Comment by chivigoku** [comment]
Hi @ddurdle,

Here is an example:
When I create a Strm with GDrive Addon, this is what I get into the STRM itself

plugin://plugin.video.gdrive/?mode=video&username=test&filename=1iAc6vFf2IKJROfSvP1pdQ-YDq6h&title=Aquaman+%282018%29.mkv

And with the Web version, I am able to create an STRM like 

http://localhost:9988/default/?mode=video&username=test&filename=1iAc6vFf2IKJROfSvP1pdQ-YDq6h&title=Aquaman+%282018%29.mkv


So my question is, Can GDrive Addon from Kodi create the same Strm´s as Web version does?

Thanks and regards

**Comment by bnichs5** [comment]
I think I already asked this question or a similar one. It's complicated
and I can't be the one to answer this. But I'm fairly certain that the
movie that you are streaming from the .strm is playing through your ram.
Not entirely sure if that can then be directed to emby to be used in a
second layer stream and especially still have control over playback. TBH
I'm not even sure if you can understand what I'm saying haha.

**Comment by chivigoku** [comment]
Hi @bnichs5,

I can figure out what you are saying but is something doesn’t make sense with what Im asking (no ofense) 

Once think is run a video using RAM, for the buffer for example, and other thing is link the movie to a plugin (as GDrive Addon does) or link via http (as web version does). 

According to @ddurdle, the web version is like a fork of the plugin one but I don’t know if the Addon itself is able to create Strm that can be played in Emby servers and then playback with any emby client or HTML5 tool (as web version)

Hope I’m clear in what I’m looking for :D

Cheers

**Comment by chivigoku** [comment]
Hi @ddurdle,

Let me attach an image to be more clear.
Here is the option to rename Strm path to "http" (red mark) in the video_stream one:

<a href="https://ibb.co/gvjScyf"><img src="https://i.ibb.co/SQstZNM/Capture.png" alt="Capture" border="0"></a><br /><a target='_blank' href='https://es.imgbb.com/'>picture upload</a><br />


Is there this option available in GDrive Kodi Addon?
If yes, then Emby-Server in my Android TV will be able to playback files from those Strm, I guess.

I´ve seen when creating strm in the web version, in order to playback videos, I do need to keep running default.py and this is who search on Google API the movie and shows the movie in Emby Client, so here I don´t know if GDrive Addon in Kodi has to be open during playback (as long as this option I mentioned above is possible).

Thanks

**Comment by hackmonker** [comment]
I have been struggling with this same issue but in my case it is not emby but jellyfin. My kodi setup works perfectly but unable to make my jellyfin setup work.`
